libpromises/actuator.c

```diff
@@ -27,6 +27,9 @@
 
 PromiseResult PromiseResultUpdate(PromiseResult prior, PromiseResult evidence)
 {
+    assert(prior != 0);
+    assert(evidence != 0);
```

Comment:
Now these asserts are really confusing without any comment.

Reply:
Removed it now, shouldn't be necessary.